numba/cpython/builtins.py

```diff
@@ -84,7 +103,8 @@ def const_eq_impl(context, builder, sig, args):
     return impl_ret_untracked(context, builder, sig.return_type, res)
 
 
-@lower_builtin(operator.ne, types.StringLiteral, types.StringLiteral)
```

Comment:
Why is this deleted?

Reply:
This is only to have the same specifications for `eq `and `ne`. Note it's not removed but changed to:
`@lower_builtin(operator.ne, types.Literal, types.Literal)`
which will work for StringLiteral types as well.

Reply:
I see, thanks, guess this https://github.com/numba/numba/pull/6015/files#diff-87f8c6b1645b5d35540368237bcc89baR107 is due to https://github.com/numba/numba/pull/6015/files#diff-87f8c6b1645b5d35540368237bcc89baR94? Should probably add a similar note?

Reply:
Yes, just added it.

Reply:
Thanks!